shared/router-v2/account-switcher/index.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import HeaderHOC from '../../common-adapters/header-hoc'
+import * as ConfigTypes from '../../constants/types/config'
+import {Props as HeaderHocProps} from '../../common-adapters/header-hoc/types'
+
+export type AccountRowItem = {
+  account: ConfigTypes.ConfiguredAccount
+  fullName: string
+}
+export type RowsProps = {}
+
+export type Props = {
+  accountRows: Array<AccountRowItem>
+  fullname: string
+  onAddAccount: () => void
+  onCancel: () => void
+  onCreateAccount: () => void
+  onProfileClick: () => void
+  onSelectAccount: (username: string) => void
+  username: string
+} & HeaderHocProps
+
+const MobileHeader = (props: Props) => (
+  <>
+    <Kb.Box2 direction="vertical" gap="tiny" gapStart={true} centerChildren={true} gapEnd={true}>
+      <Kb.Avatar username={props.username} onClick={props.onProfileClick} size={128} />
+      <Kb.Box2 direction="vertical" centerChildren={true}>
+        <Kb.Text type="BodyBig" onClick={props.onProfileClick}>
+          {props.username}
+        </Kb.Text>
+        <Kb.Text type="BodySmall" lineClamp={1} onClick={props.onProfileClick}>
+          {props.fullname}
+        </Kb.Text>
+      </Kb.Box2>
+    </Kb.Box2>
+    <Kb.Box2 direction="vertical" style={styles.buttonBox} fullWidth={true} gap="tiny">
+      <Kb.Button onClick={props.onAddAccount} label="Log in as another user" mode="Primary" fullWidth={true} />
+      <Kb.Button
+        onClick={props.onCreateAccount}
+        label="Create a new account"
+        mode="Secondary"
+        fullWidth={true}
+      />
+    </Kb.Box2>
+  </>
+)
+const AccountsRows = (props: Props) => (
+  <Kb.Box2 direction="vertical" fullWidth={true}>
+    {props.accountRows.map(entry => (
+      <Kb.ListItem2
+        type="Small"
+        icon={
+          <Kb.Avatar
+            size={32}
+            username={entry.account.username}
+            style={!entry.account.hasStoredSecret && styles.avatarSignedOut}
```

Comment:
this should be s ternary, style can't be false. this is being fixed in my strict branch